azure_rm_aks/create_scale_aks.yml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: localhost
+  roles:
+    - Azure.azure_preview_modules 
+  vars:
+    resource_group: ansible-aks
+    name: ansible-aks
+    location: eastus
+    AZURE_CLIENT_ID: your-client-id
+    AZURE_SECRET: your-secret
+    SSH_PUBLIC_KEY: your-ssh-public-key
+  tasks:
+  - name: Create a resource group
+    azure_rm_resourcegroup:
+     name: "{{ resource_group }}"
+     location: "{{ location }}"
+  - name: Create an AKS instance
+    azure_rm_aks:
+     name: "{{ name }}"
+     resource_group: "{{ resource_group }}"
+     location: eastus
+     dns_prefix: "{{ name }}"
+     kubernetes_version: 1.8.1
```

Comment:
update this to variable, and default to 1.7.9 to keep consistent with az cli default.

Reply:
at least AKS created by az cli is 1.7.9. since version is one of parameter, so it should be configurable.